package.json

```diff
@@ -37,6 +43,7 @@
     "build:examples": "webpack --config webpack.example.config.js --sort-modules-by size --define process.env.NODE_ENV=production",
     "build:dist": "webpack --config webpack.config.js --display-modules --sort-modules-by size --define process.env.NODE_ENV=production",
     "build:es5": "babel src --out-dir ./lib/es5/ --presets babel-preset-env --plugins babel-plugin-transform-runtime",
-    "coverage": "istanbul cover ./node_modules/mocha/bin/_mocha"
+    "coverage": "istanbul cover ./node_modules/mocha/bin/_mocha",
+    "lint": "eslint \"src/**/*.js\" \"test/**/*.js\" \"benchmarks/**/*.js\""
```

Comment:
Is this the easiest way? Doesn't standard itself have an executable?

Reply:
I'm not sure, standard does have an executable but here we're using eslint for that and I've only added `eslint-config-standard` as a devDependency not the standard executable :)